src/pip/_internal/commands/show.py

```diff
@@ -29,7 +31,9 @@ class ShowCommand(Command):
     ignore_require_venv = True
 
     def __init__(self, *args, **kw):
-        super(ShowCommand, self).__init__(*args, **kw)
+        # type: (*Any, **Any) -> None
+        # https://github.com/python/mypy/issues/4335
+        super(ShowCommand, self).__init__(*args, **kw)  # type: ignore
```

Comment:
Similarly `# type: ignore` can be removed from here as well.

Reply:
Fixed in latest commit.